cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/task/ClouderaManagerTemplateInstallationChecker.java

```diff
@@ -92,4 +62,21 @@ public String successMessage(ClouderaManagerCommandPollerObject clouderaManagerC
     protected String getCommandName() {
         return "Template install";
     }
+
+    private void fail(String messagePrefix, ApiCommand apiCommand, CommandsResourceApi commandsResourceApi) {
+        List<String> errorReasons = ClouderaManagerCommandApiErrorParserUtil.getErrors(apiCommand, commandsResourceApi);
+        String msg = messagePrefix + "Installation of CDP with Cloudera Manager has failed: [" + String.join(", ", errorReasons) + "]";
+        LOGGER.info(msg);
+        for (String errorReason : errorReasons) {
+            // Unfortunately CM is not giving back too much details about the errors, and what is returned usually nondeterministic:
+            // In a good case it returns "Failed to create HDFS directory",
+            // but sometines it just returns "Aborted command" or "Command timed-out after 186 seconds", so matching on such generic error error messages
+            // has no added value, therefore we are just checing whether AudiTDir related commands are failing or not.
+            if (errorReason.matches(".*[RangerPluginCreateAuditDir|CreateRangerAuditDir|CreateRangerKafkaPluginAuditDirCommand|" +
+                    "CreateHiveWarehouseExternalDir|CreateHiveWarehouseDir|CreateRangerKnoxPluginAuditDirCommand].*")) {
```

Comment:
There are some more service-specific audit dir commands, see [FirstPartyCsdServiceTypes in CM](https://github.infra.cloudera.com/Starship/cmf/blob/cdpd-master/web/src/main/java/com/cloudera/cmf/service/csd/components/FirstPartyCsdServiceTypes.java#L79-L91). I would include all of them, even if e.g. Ranger KMS is currently not part of CDP PC. (This might change later if we need to manage custom data encryption keys inside clusters.)